src/uct/ib/dc/dc_mlx5.c

```diff
@@ -1254,6 +1257,46 @@ static uct_iface_ops_t uct_dc_mlx5_iface_tl_ops = {
     .iface_get_address        = uct_dc_mlx5_iface_get_address,
 };
 
+static ucs_status_t uct_dc_mlx5dv_calc_tx_wqe_ratio(uct_ib_mlx5_md_t *md)
+{
+    uct_ib_device_t *dev               = &md->super.dev;
+    uct_ib_qp_init_attr_t qp_init_attr = {};
+    struct mlx5dv_qp_init_attr dv_attr = {};
+    struct ibv_qp *dci_qp;
+    ucs_status_t status;
+    uct_ib_mlx5dv_qp_init_objs_t qp_init_objs;
+
+    status = uct_ib_mlx5dv_qp_init_objs_create(&md->super, &qp_init_objs);
+    if (status != UCS_OK) {
+        goto out;
+    }
+
+    uct_ib_mlx5dv_qp_init_attr(&qp_init_attr, md->super.pd, &qp_init_objs,
+                               IBV_QPT_DRIVER, 0);
+    uct_ib_mlx5_dci_qp_update_attr(&qp_init_attr);
+    uct_ib_mlx5dv_dc_qp_init_attr(&dv_attr, MLX5DV_DCTYPE_DCI);
+
+    dci_qp = UCS_PROFILE_CALL_ALWAYS(mlx5dv_create_qp, dev->ibv_context,
```

Comment:
no need to profile this call, it is not a data path

Reply:
it was added here - https://github.com/openucx/ucx/pull/8184.
i guess the idea is to count how many QPs are created